examples/lstv-based-vad/ARMA.py

```diff
@@ -0,0 +1,41 @@
+from __future__ import division
+import numpy as np
+from scipy.io import wavfile
+from scipy import signal
+import matplotlib.pyplot as plt
+import matplotlib.mlab as mlb
+
```

Comment:
https://github.com/pykaldi/pykaldi/blob/8af7a10f9978c0acaae15b8dc0f660d5f9be1da7/examples/lstv-based-vad/ARMA.py#L3-L6

Remove these 4 imports. They are not used in this file. This comment applies to a number of files in this PR so please update others too.